backend/order.js

```diff
@@ -243,11 +268,11 @@ var order = {
     }
   },
 
-  paymentDone: function(order_id, params, cb) {
+  paymentDone: function(order_id, params, skipVerification, cb) {
```

Comment:
I feel a bit bad about these dangerous boolean arguments.. Could we have paymentDone(order_id, params, cb) which just calls this with skipVerification = false? (Effectively a default value for an arg.) Or include skipVerification in params - it'd be clear that this "true" will skip verification?